_posts/en/pages/2024-06-26-disclosures.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Disclosures
```

Comment:
nit: I think "Security Advisories" would be the better title since it is widely established terminology:

* https://www.mozilla.org/en-US/security/advisories/
* https://www.vmware.com/security/advisories.html
* https://sec.cloudapps.cisco.com/security/center/publicationListing.x
* https://www.dell.com/support/security/en-ca

Reply:
Changed.